src/neo-vm/ExecutionEngine.cs

```diff
@@ -558,6 +552,54 @@ private bool ExecuteInstruction()
                     }
 
                 // Splice
+                case OpCode.NEWBUFFER:
+                    {
+                        if (!TryPop(out PrimitiveType item_n)) return false;
+                        BigInteger n = item_n.ToBigInteger();
+                        if (n < 0 || n > MaxItemSize) return false;
+                        Push(new byte[(int)n]);
```

Comment:
I think that it could be very expensive to create a buffer like this

```csharp
var data = new byte [512]
for (int x =0; x<512; x++)
 data [x] =x;
```

Reply:
Do you have any idea?

Reply:
With inmutable stack items... no, maybe we can have a memory buffer, where you can copy there? like we have Slots, a memory store?

Reply:
Another idea is to move `ByteArray` from `PrimitiveType` to `CompoundType`.

Reply:
I will remove the 3 new opcodes first.